libs/database/src/lib/models/user.model.ts

```diff
@@ -68,3 +68,15 @@ export interface UserProfile {
   details?: string;
   images?: UserImage;
 }
+
+export interface SuggestionWords {
+  SWEARWORDS: string[];
+  SUBJECTS: string[];
+  ADJECTIVES: string[];
+  FORBIDWORDS: {
+    HIGH: string[];
+    LOW: string[];
+  };
+  LENGTH_MAX: number;
+  LENGTH_MIN: number;
+}
```

Comment:
```suggestion
export interface CastcleIdMetadata {
  minLength: number;
  maxLength: number;
  bannedWords: string[];
  nouns: string[];
  adjectives: string[];
}
```
รวมพวกคำที่ใช้ไม่ได้มาอยู่ใน bannedWords เลย